src/components/InsightsData/useStyles.js

```diff
@@ -0,0 +1,42 @@
+import { makeStyles } from "@material-ui/core/styles";
+
+const useStyles = makeStyles(({ typography, breakpoints }) => ({
+  root: {},
+  section: {
+    paddingTop: typography.pxToRem(60),
+    "& .highlight": {
+      background:
+        "linear-gradient(180deg,rgba(255,255,255,0) 30%, #0067A31A 50% )",
+      display: "inline-block",
+    },
```

Comment:
Why have this on section instead of subtitle?

Reply:
Fixed